src/sync/app.hpp

```diff
@@ -138,7 +138,10 @@ class App {
         void disable_api_key(const UserAPIKey& api_key,
                              std::function<void(Optional<AppError>)> completion_block);
     private:
-        UserAPIKeyProviderClient(App*);
+        friend class App;
+        UserAPIKeyProviderClient(App* app) : parent(app), m_provider_key("api_keys") {}
+        App* parent;
+        std::string m_provider_key;
```

Comment:
Why is this a member variable? It appears to just be a constant?

Reply:
Noted, changed to constant